src/platforms/ctxlink/WiFi_Server.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
```

Comment:
This gets the same include guards and copyright notice comment as that other header please.